lib/windows/winlean.nim

```diff
@@ -36,6 +36,8 @@ type
   DWORD* = int32
   PDWORD* = ptr DWORD
   LPINT* = ptr int32
+  ULONG_PTR* = uint
```

Comment:
I'm inclined to agree with cheatfate. Wrapped procedures and structures need to mirror their C counterparts as closely as possible, otherwise we get errors.